src/content/time-tracking.mdx

```diff
@@ -0,0 +1,65 @@
+# Time Tracking Guide
+
+This tracking guide is not the same as general company’s rules. It relates only developer stuff and additional things which can be helpful for our work.
+For more projects/tasks details, please read our company tracking rules.
+
+## Where should I track my time?
+We are using a Harvest app to store tracked time. You can use a web / phone / desktop app or Chrome / Mozilla / Microsoft Edge extension [(look here)](https://www.getharvest.com/apps-and-integrations/directory#mobile-desktop). You can choose which one will be the best for you, to make it as easy as it’s possible & not disturb you while your daily stuff.
+
+
+## How should I track?
+
+### Project’s tasks
+It was intentionally described „Harvest app to store tracked time”, because it’s not always a place where you should add tasks.
```

Comment:
```suggestion
It was intentionally described as „Harvest app to store tracked time”, because it’s not always the place where you should add tasks.
```

Reply:
This tense has no sense after this change https://github.com/Appnroll/appnroll-developers/pull/35#discussion_r459406798 😞

Reply:
I made an update https://github.com/Appnroll/appnroll-developers/pull/35/commits/56e09b94bb9b0f47a195ceffb587cd3e5666ceb4